From 4924b7b5517c9c334cf5faa3c7a29adf9a0c0ba1 Mon Sep 17 00:00:00 2001
From: Kieran Bingham <kieran.bingham@ideasonboard.com>
Date: Wed, 28 Aug 2019 15:54:19 +0100
Subject: [PATCH 787/806] media: bcm2835: unicam: Reduce scope of local
 function

unicam_start_rx() is not used outside of the unicam module.  Its current
definition produces a compiler warning, that no function prototype
exists.

As the function is only used within the local scope of the module,
convert it to a static function.

Signed-off-by: Kieran Bingham <kieran.bingham@ideasonboard.com>
---
 drivers/media/platform/bcm2835/bcm2835-unicam.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/media/platform/bcm2835/bcm2835-unicam.c
+++ b/drivers/media/platform/bcm2835/bcm2835-unicam.c
@@ -963,7 +963,7 @@ static void unicam_cfg_image_id(struct u
 	}
 }
 
-void unicam_start_rx(struct unicam_device *dev, unsigned long addr)
+static void unicam_start_rx(struct unicam_device *dev, unsigned long addr)
 {
 	struct unicam_cfg *cfg = &dev->cfg;
 	int line_int_freq = dev->v_fmt.fmt.pix.height >> 2;
